docs/config/configuration.md

```diff
@@ -239,6 +239,8 @@ bitcoin-s {
 }
 ```
 
+The database driver will create a separate SQL namespace for each sub-project: `chain`, `node` and `wallet`.
```

Comment:
I think more on this page needs to be updated

https://bitcoin-s.org/docs/next/config/configuration#database-configuration

For example, do you need `username` and `password` for each module in the configuration now? 

Can't you just say 

```
bitcoin-s.chain.profile = bitcoin-s.common.profile
bitcoin-s.chain.db = bitcoin-s.common.db
bitcoin-s.wallet.profile = bitcoin-s.common.profile
bitcoin-s.wallet.db = bitcoin-s.common.db 
```

now?